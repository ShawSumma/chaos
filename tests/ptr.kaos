ptr a = 0x7fffb1d7ec80
print a
