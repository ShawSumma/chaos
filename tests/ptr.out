0x00007fffb1d7ec80
